homework-submission/homeworkWeek1/main.jsx

```diff
@@ -0,0 +1,118 @@
+'use strict';
+
+const { Component } = React;
+
+// This component renders list items
+const ListItem = props => {
+  const { listData, isStatic, markAsCompleted } = props;
+  let { description, deadline, id, done } = listData;
+  return (
+    <li
+      id={id}
+      onClick={() => {
+        if (isStatic) {
+          const element = document.getElementById(id);
+          element.classList.toggle('done');
+        } else {
+          markAsCompleted(id - 1);
+        }
+      }}
+      className={done ? 'done' : 'not-done'}
+    >
+      {description},{deadline}
+    </li>
+  );
+};
+
+//This component creates a static list and displays listItem components
+const StaticList = props => {
+  const { isStatic } = props;
+  return (
+    <div>
+      <h2>Static List</h2>
+      <ul className="static list">
+        <ListItem
+          listData={{ id: 1, description: 'Get out of bed', deadline: '13.9.2017' }}
+          isStatic={isStatic}
```

Comment:
Even if we are going to remove this one `isStatic` but just I liked your way of making things separated and ordered nicely  💯 

Mark This as **Second Step**